charts/cluster-autoscaler/templates/prometheusrule.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.prometheusRule.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "cluster-autoscaler.fullname" . }}
+  {{- if .Values.prometheusRule.namespace }}
+  namespace: {{ .Values.prometheusRule.namespace }}
+  {{- end }}
+  labels: {{- toYaml .Values.prometheusRule.additionalLabels | nindent 4 }}
+spec:
+  groups:
+    - name: {{ include "cluster-autoscaler.fullname" . }}
+      rules: {{- tpl (toYaml .Values.prometheusRule.rules) . | nindent 8 }}
```

Comment:
Do we want to support users setting the interval as well? (Potentially the partial_response_strategy as well, though that's probably less useful, so not as sold on including.) 

https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#rulegroup

Reply:
Good catch ! , i'll add that.

Reply:
Done for `interval`. Not sure about `partial_response_strategy` either.